Bottom: 20d0bff459590313535e98a0130a647140ecec97
Top:    91bc251274c1981812043d3cd8e2e72a431611d3
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-25 12:28:44 -0700

Refresh of rs-opt-inline

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 7802ef5..5845ca9 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -105,7 +105,7 @@ struct rs_sge {
 	uint32_t length;
 };
 
-#define RS_INLINE_MIN    (sizeof(struct rs_sge))
+#define RS_MIN_INLINE    (sizeof(struct rs_sge))
 #define rs_host_is_net() (1 == htonl(1))
 #define RS_CONN_FLAG_NET 1
 
@@ -1716,8 +1716,8 @@ int rsetsockopt(int socket, int level, int optname,
 			break;
 		case RDMA_INLINE:
 			rs->sq_inline = min(*(uint32_t *) optval, RS_QP_MAX_SIZE);
-			if (rs->sq_inline < RS_INLINE_MIN)
-				rs->sq_inline = RS_INLINE_MIN;
+			if (rs->sq_inline < RS_MIN_INLINE)
+				rs->sq_inline = RS_MIN_INLINE;
 			break;
 		default:
 			break;
